src/framework/extensions/view/extensionsmodel.cpp

```diff
@@ -150,6 +150,7 @@ void ExtensionsListModel::editShortcut(QString codeKey)
 void ExtensionsListModel::reloadPlugins()
 {
     provider()->reloadPlugins();
+    load();
```

Comment:
better make like this: 
```
ExtensionsListModel::ExtensionsListModel(QObject* parent)
    : QAbstractListModel(parent)
{
    ...

    provider()->manifestListChanged().onNotify(this, [this]() {
        load();
    });
}
```

Reply:
> Yeah I had wondered that, but it seemed to work fine as is.

the subscription option is better because it will work if the plugin reload is not caused by this model, but for some other reason (shortcut, some API, some request from the server, etc.).

> BTW do you want me to change the file name to match the class? (extensionsListModel?)

that would be nice, thank you!

Reply:
Ok, have put rename into a separate commit